examples/photogrammetry/photogrammetryAPI.ipynb

```diff
@@ -35,7 +21,10 @@
     "\n",
     "Revisions can for instance represent time evolution of the 3D model, or different versions of the model (high detail vs low detail).\n",
     "\n",
-    "When you create a new revision and upload images, the reconstruction will start immediately. Depending on the the quality and number of images, this can take some time. A revision can have status `CREATED`, `IN_PROGRESS`, `SUCCESS` or `FAILED`, which can be tracked during processing."
+    "When you create a new revision and upload images, the reconstruction will start immediately. Depending on the the quality and number of images, this can take some time. A revision can have status `CREATED`, `IN_PROGRESS`, `SUCCESS` or `FAILED`, which can be tracked during processing.\n",
+    "\n",
+    "\n",
+    "The API reference is available here: [https://doc.cognitedata.com/api/playground/](https://doc.cognitedata.com/api/playground/)"
```

Comment:
Maybe link directly to the photogrammetry part of playground documentation.
https://docs.cognite.com/api/playground/#tag/Photogrammetry